hs/src/Reach/Verify/SMT.hs

```diff
@@ -1050,12 +1050,12 @@ _verify_smt mc vst smt lp = do
   -- something reasonable, like 64-bit?
   let defineIE who (v, it) =
         case it of
-          T_Fun {} -> mempty
+          -- ST_Fun {} -> mempty
           _ ->
             pathAddUnbound_v ctxt Nothing at (smtInteract ctxt who v) it O_Interact
   let definePIE (who, InteractEnv iem) = do
         pathAddUnbound_v ctxt Nothing at (smtAddress who) T_Address O_BuiltIn
-        mapM_ (defineIE who) $ M.toList iem
+        mapM_ (defineIE who) $ M.toList $ M.map st2dt iem
```

Comment:
This is what I was mentioning before... no `SLType` should be exposed past `Eval.hs`